src/hooks/useGardens.tsx

```diff
@@ -49,7 +49,46 @@ const useGardens = () => {
     return execute(config)
   }
 
-  return { getGardens, getGarden, syncUsers, deleteGarden }
+  const createGarden = (gardenName: string, data: unknown) => {
```

Comment:
Can you add a comment explaining why this has a `gardenName` argument that isn't used? (I know why; I'm asking for future me who will forget why... ;-))